resources/uk/gov/hmcts/gradle/init.gradle

```
@@ -38,11 +38,21 @@ settingsEvaluated { settings ->
   }
 }
 
-// Enable JSON dependency check reports for monitoring of CVEs and suppressions.
 allprojects {
   afterEvaluate { project ->
     if (project.hasProperty('dependencyCheck')) {
+      // Any CVE should fail the build.
+      project.dependencyCheck.failBuildOnCVSS = 0
+
+      // JSON dependency check reports are logged to cosmos for reporting.
       project.dependencyCheck.formats += 'json'
+
+      // Scan only the dependencies used at application runtime.
+      // See https://docs.gradle.org/current/userguide/java_plugin.html#sec:java_plugin_and_dependency_management
+      project.dependencyCheck.scanConfigurations += 'runtimeClasspath'
```

this would be more obvious to me, I had to go check the docs to understand why appending one worked, ([] implies all configuration).

```suggestion
      project.dependencyCheck.scanConfigurations = ['runtimeClasspath']
```

---

I made it additive so that any existing configurations are respected and not cleared out.

---

(since this is executing after build configuration)

---

is it an issue if someone already has `runtimeClasspath` in their `scanConfigurations`?

---

The dependency checker just checks that list for existence so duplicates should be no problem. I've added a check nevertheless.